### ISSUE TITLE
Anyone tried installing this Docker recently? Nothing happens after clicking install and clicking Done

### ISSUE THREAD

**Issue by DigaumSpider** [imthenachoman/unraid-templates#10]
Anyone tried installing this Docker recently in Unraid?
Nothing happens after clicking install and clicking Done

I click the Install button on the apps page, get taken to the configuration page, then after making sure it's all good, I click Done and, instead of seeing this show up running under my docker images, I'm taken back to the apps page.

Is this expected? Should I be changing anything else in the settings before clicking Done?

Thanks

**Comment by tatezach** [comment]
Yes, installed it today without a problem. Maybe check your Unraid syslog.

**Comment by AlanH260** [comment]
I am also having problems with this docker.  It says it has installed but it is unhealthy.  

curl ifconfig.io returns the following.

root@0c35387019e3:/# curl ifconfig.io
curl: (6) Could not resolve host: ifconfig.io
root@0c35387019e3:/# 

Here is a copy of the log file.  I am a complete newbie at this and cannot figure out where I am going wrong.  

[#] 
[#] ip link add wg0 type wireguard
[#] wg setconf wg0 /dev/fd/63
[#] ip -4 address add 10.5.0.2/32 dev wg0
[#] ip link set mtu 1420 up dev wg0
[#] resolvconf -a wg0 -m 0 -x
[#] wg set wg0 fwmark 51820
[#] ip -4 route add 0.0.0.0/0 dev wg0 table 51820
[#] ip -4 rule add not fwmark 51820 table 51820
[#] ip -4 rule add table main suppress_prefixlength 0
[#] iptables-restore -n
[#] 
      }
    }
  ],
  "groups": [
    {
      "id": 11,
      "created_at": "2017-06-13 13:43:00",
      "updated_at": "2017-06-13 13:43:00",
      "title": "Standard VPN servers",
      "identifier": "legacy_standard",
      "type": {
        "id": 3,
        "created_at": "2017-06-13 13:40:17",
        "updated_at": "2017-06-13 13:40:23",
        "title": "Legacy category",
        "identifier": "legacy_group_category"
      }
    },
    {
      "id": 15,
      "created_at": "2017-06-13 13:43:38",
      "updated_at": "2017-06-13 13:43:38",
      "title": "P2P",
      "identifier": "legacy_p2p",
      "type": {
        "id": 3,
        "created_at": "2017-06-13 13:40:17",
        "updated_at": "2017-06-13 13:40:23",
        "title": "Legacy category",
        "identifier": "legacy_group_category"
      }
    },
    {
      "id": 21,
      "created_at": "2017-10-27 14:23:03",
      "updated_at": "2017-10-30 08:09:48",
      "title": "The Americas",
      "identifier": "the_americas",
      "type": {
        "id": 5,
        "created_at": "2017-10-27 14:16:30",
        "updated_at": "2017-10-27 14:16:30",
        "title": "Regions",
        "identifier": "regions"
      }
    }
  ],
  "specifications": [
    {
      "id": 8,
      "title": "Version",
      "identifier": "version",
      "values": [
        {
          "id": 257,
          "value": "2.1.0"
        }
      ]
    }
  ],
  "ips": [
    {
      "id": 560429,
      "created_at": "2022-05-25 11:01:28",
      "updated_at": "2022-05-25 11:01:28",
      "server_id": 989912,
      "ip_id": 9716390,
      "type": "entry",
      "ip": {
        "id": 9716390,
        "ip": "181.214.70.34",
        "version": 4
      }
    }
  ]
}
[2023-04-17T05:38:08-07:00] Connecting...
[2023-04-17T05:38:09-07:00] Connected! \(ᵔᵕᵔ)/

Thanks,

**Comment by imthenachoman** [comment]
I've been having issues with the NordLynx containers. I haven't been able to figure it out. :/

**Comment by WeezyMike** [comment]
Cant get key to pull anymore, just keep getting back this...


docker run --rm --cap-add=NET_ADMIN -e USER=XXX -e PASS=YYY bubuntux/nordvpn:get_private_key
Waiting for daemon to start up...
A new version of NordVPN is available! Please update the application.
Token parameter value is missing.
Invalid token. ¯\_(ツ)_/¯
